### DISCUSSION TITLE
d-cite in page layout

### DISCUSSION THREAD

**Discussion by ninyam** [alshedivat/al-folio#2387] [Q&A] [answered]
Hi there!

I'd like to add a citation using `<d-cite key=""></d-cite>`, but I want to do it a page layout. Is that possible? If so, how can I do it?

Thank you!

**Comment by george-gca** [comment, marked as answer]
If you want to do citations and create a bibliography, I'd recommend using the citations from [jekyll scholar](https://github.com/inukshuk/jekyll-scholar?tab=readme-ov-file#citations).
